cypress/integration/loggingIn.js

```diff
@@ -1,48 +1,100 @@
-describe("Logging in", () => {
-  describe("Given a valid ward code", () => {
-    before(() => {
-      // reset and seed the database
-      cy.exec(
-        "npm run dbmigratetest reset && npm run dbmigratetest up && npm run db:seed"
-      );
-    });
-    it("when a user enters this information, the user should be able to sign in", () => {
-      cy.visit(Cypress.env("baseUrl"));
-      cy.get("input").type(Cypress.env("validWard"));
-      cy.get("button").contains("Log in").click();
-      cy.contains("The code you entered was not recognised").should(
-        "not.be.visible"
-      );
-      cy.contains("Book a virtual visit").should("be.visible");
-    });
-
-    it("when a trust admin enters this information, the user should be able to sign in", () => {
-      cy.visit(Cypress.env("baseUrl"));
-      cy.get("input").type(Cypress.env("validTrustAdminCode"));
-      cy.get("button").contains("Log in").click();
-      cy.contains("The code you entered was not recognised").should(
-        "not.be.visible"
-      );
-      cy.contains("Test Trust").should("be.visible");
-      cy.contains("Dashboard").should("be.visible");
-    });
-
-    it("when an admin enters this information, the user should be able to sign in", () => {
-      cy.visit(Cypress.env("baseUrl"));
-      cy.get("input").type(Cypress.env("validAdminCode"));
-      cy.get("button").contains("Log in").click();
-      cy.contains("The code you entered was not recognised").should(
-        "not.be.visible"
-      );
-      cy.contains("Site administration").should("be.visible");
-    });
+describe("As a user, I want to log in so that I can access the service.", () => {
+  before(() => {
+    // reset and seed the database
+    cy.exec(
+      "npm run dbmigratetest reset && npm run dbmigratetest up && npm run db:seed"
+    );
   });
 
-  it("Given an invalid code, when a user enters this informtion, the user should see an error", () => {
+  it("allows a ward staff to log in and displays their home page", () => {
+    GivenIAmAWardStaff();
```

Comment:
Is this line required (given the function is empty) ?

Reply:
No, it's not required. It's there for readability. If preferred, I can remove?

Reply:
It keeps things consistent so I'm ok with leaving it in